src/Compilers/CSharp/Test/Syntax/Parsing/RadOnlyStructs.cs

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Xunit;
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit.Abstractions;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Parsing
+{
+    [CompilerTrait(CompilerFeature.ReadonlyReferences)]
```

Comment:
Will fix the trait to be "ReadOnlyReferences"

---
In reply to: [118027328](https://github.com/dotnet/roslyn/pull/19713#discussion_r118027328) [](ancestors = 118027328)